src/main/resources/db/migration/V008__Populate_status_column.sql

```diff
@@ -0,0 +1,13 @@
+UPDATE
+  letters
+SET
+  status = (
+    CASE
+      WHEN sent_to_print_at IS NULL THEN
```

Comment:
As discussed in the migration conversation yesterday we could set the status to 'Uploaded' (since they're on the service bus) instead of 'Created' and collapse these two cases.

This ensures the new scheduled tasks will not attempt to reupload letters in the created state that have no pdf. If the letters don't upload then our failed letters task will alert us to them.

Reply:
Done